_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+ [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) is required to interact with Amazon SES.
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.
+ [Configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) basic settings used by AWS CLI.
+- SES transport requires the `ses:SendRawEmail` role.
+
+```
+{
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "ses:SendRawEmail",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+The following command requests an email with the report attached:
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport ses --from <sender_email_id> --to <recipient_email_id>
+```
+
+The following command uses default values for all other options. You can also set `FROM`, `TO`, and `TRANSPORT` in .env file and use following command.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d
+```
+
+To modify the body of your email, you can simply edit *index.hbs* file.
+
+### Example: Send a report to email with SMTP
+
+To send a report through email with SMTP transport, you need to set the options `SMTP_HOST`, `SMTP_PORT`, `SMTP_USER`, `SMTP_PASSWORD`, `SMTP_SECURE` in your .env file.
+
+Once the transport options are set in .env file, you can send the email using the following command.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport smtp --from <sender_email_id> --to <recipient_email_id>
+```
+
+You can choose to set options using *.env* file or the command line argument values in any combination. Make sure to specify all required values to avoid getting errors.
+
+To modify the body of your email, you can simply edit *index.hbs* file.
+
+## Environment variable limitations
+
+The following limitations apply to environment variable usage with the Reporting CLI:
+
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.
+ Supported platforms are Windows x86, Windows x64, Mac Intel, Mac ARM, Linux x86, Linux x64.
+  
+  For any other platform, users can take advantage of *CHROMIUM_PATH* environment variable to use custom chromium.
+
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.
+
+  bash: `set +H`
+
+  zsh: `setopt nobanghist`
+
+
+  Alternatively, you can add a URL value as an environment variable with the following format `URL="<url-with-!>"`.
+
+- All command options only accept lowercase letters.
+
+## Troubleshooting
+
+To resolve **MessageRejected: Email address is not verified**, see [check](https://aws.amazon.com/premiumsupport/knowledge-center/ses-554-400-message-rejected-error/) in AWS Knowledge Center.
```

Comment:
see is not required here. already has check.

Reply:
oh that's just our style guideline. 
we use 
",see [Name of page](URL)."